ci/sandbox/release-pipeline.yaml

```diff
@@ -124,10 +129,8 @@ spec:
           - get: vmc-image
             passed: [build]
             trigger: true
-            params: {skip_download: true}
           - get: cloudhsm-image
```

Comment:
does this image ever get built? :thinking: 

I think there is a missing build-step here ... I suspect the old pipeline was pulling an image build from another project's pipeline (proxy-node?).

I think we need to add a build step for it here really, it's a bit weird to be stealing the image from another project (even if they are somewhat intertwined)

Reply:
It seems you're right - the verify-proxy-node pipeline put to a cloudhsm-client-image resource that had a repository that matches the one here. The two tenants have separate image repositories now so we will need to build it separately.